### DISCUSSION TITLE
backdrop-blur doesn't work in Tailwind Play

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#5200] [Help]
I was following a tailwind lab youtube channel video to recreate iOS lockscreen, but when I tried to make summary panel blury it doesn't work, although I did use `backdrop-filter` with `backdrop-blur-3xl` , here is my code below
https://play.tailwindcss.com/ozub7LbuNT?size=540x720

**Comment by Martleby** [comment]
It works for me in Chrome, but I think you need to drop your bg color opacity to something smaller. Try bg-white/10 for example:

https://play.tailwindcss.com/ziSXRuKcuA?size=540x720

Cheers,
Martin.

**Reply by Chandler-Zhu** [reply]
It works in chrome but when I use firefox there is no blur effect, I don't know where the problem is, but thanks anyway
